### PR TITLE
Missing fragment in PR 778 New query fields 

### PR DESCRIPTION
**Pull request by fernandofloresg**
##### SUMMARY

Added the missing changelog in PR #778 I merged it without asking for a changelog.

##### ISSUE TYPE

- Docs Pull Request

Is a doc pull request no need to run the test suite. Notes: Squash and merge.

### CODE REVIEW

**Review by AndreMarcel99 — APPROVED** [review]
Is clear and write as should be.

File: changelogs/fragments/778-query-new-fields.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+minor_changes:
+- zos_job_query - Adds new fields job_class, svc_class, priority, asid,
+  creation_datetime, and queue_position to the return output when querying
+  or submitting a job. Available when ZOAU v1.2.3 or greater.
```
Comment (rexemin) [inline changelogs/fragments/778-query-new-fields.yml:4]: I would add `using` here: `Available when using ZOAU v1.2.3 or greater.`

**Review by rexemin — CHANGES_REQUESTED** [review]
Thanks for adding the fragment, I just made one comment.

File: changelogs/fragments/778-query-new-fields.yml (hunk 1 above)
Comment (fernandofloresg) [reply changelogs/fragments/778-query-new-fields.yml]: Agree

**Review by rexemin — APPROVED** [review]

**Pull request merged**